source/component/TopNavBar.tsx

```diff
@@ -12,14 +12,18 @@ export interface TopNavBarProps extends OffcanvasNavbarProps {
 
 const OriginalURLPattern = /^(https?|#)/;
 
+import { i18n } from '../i18n';
+
+const { t } = i18n;
+
 export const TopNavBar: FC<TopNavBarProps> = ({ menu, ...rest }) => (
```

Comment:
```suggestion
import { oberver } from 'web-cell';

export const TopNavBar: FC<TopNavBarProps> = observer(({ menu, ...rest }) => (
```
所有涉及 i18n 的组件都要变为观察者，才能相应语言切换去更新界面。

Reply:
我试了，import { observer } from 'web-cell';这个语句以后，显示上是灰暗的，看起来用不上。